apm-sniffer/config/agent.config

```diff
@@ -164,6 +164,12 @@ profile.duration=${SW_AGENT_PROFILE_DURATION:10}
 profile.dump_max_stack_depth=${SW_AGENT_PROFILE_DUMP_MAX_STACK_DEPTH:500}
 # Snapshot transport to backend buffer size
 profile.snapshot_transport_buffer_size=${SW_AGENT_PROFILE_SNAPSHOT_TRANSPORT_BUFFER_SIZE:4500}
+# If true, Async Profiler will be enabled when user creates a new async profiler task, Otherwise it is disabled.
```

Comment:
This comment is not accurate. You hard coded this value as true, so, the default value is TRUE, not disabled. Please update your default value in the codes.